docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:20.04
+
+################## METADATA ##########################
+
+LABEL base.image=ubuntu:20.04
+LABEL software="Bovine-TB Phylogeny Pipeline Image"
+LABEL about.summary="Bioinformatics Pipeline for post-processing of sequenced Bovine-TB data"
+LABEL about.documentation="https://github.com/APHA-CSU/btb-phylo"
+LABEL about.tags="Phylogenomics"
+
+################## ARGS #############################
+
+ARG BOVTB_PATH="/btb-phylo/"
```

Comment:
not sure this arg is required. This is a relic from an old messy Dockerfile from btb-seq. I reckon you just make `WORKDIR "/btb-phylo/"` a one-liner